src/components/map_v2/plugins/el-scooter-plugin/use-el-scooters-plugin.tsx

```diff
@@ -0,0 +1,46 @@
+import {isScooter, ScooterSheet, useVehicles} from '@atb/mobility';
+import {Scooters} from '@atb/components/map/components/mobility/Scooters.tsx';
+import React, {useEffect} from 'react';
+import {Feature, Point} from 'geojson';
+import {MapPluginType} from '@atb/components/map_v2/plugins/types.ts';
+import {useBottomSheet} from '@atb/components/bottom-sheet';
+
+export const useElScootersPlugin: MapPluginType = ({
+  region,
+  setSharedState,
+}) => {
+  const {open: openBottomSheet, close: closeBottomSheet} = useBottomSheet();
+  const vehicles = useVehicles({SCOOTER: {showAll: true, operators: []}});
+  const elScooterFeatures = vehicles?.vehicles.scooters;
+  const updateRegion = vehicles?.updateRegion;
+
+  useEffect(() => {
+    if (region) updateRegion?.(region);
+  }, [region, updateRegion]);
```

Comment:
After the tile server + vector source is in place, we can completely remove the updateRegion stuff :)